blueflood-core/src/integration-test/java/com/rackspacecloud/blueflood/io/astyanax/ACassandraUtilsIO.java

```diff
@@ -12,7 +13,7 @@
 /**
  * Utility class to provide some test methods existing tests use.
  */
-public class AstyanaxCassandraUtilsIO implements CassandraUtilsIO {
+public class ACassandraUtilsIO implements CassandraUtilsIO {
 
```

Comment:
Its to shorten the name from AstyanaxXXX to AXXX. We have Datastax equivalent of most of these classes too. But eventually, the Astyanax classes are going to be removed.